python_modules/dagster/dagster/_utils/yaml_utils.py

```diff
@@ -125,7 +125,7 @@ def merge_yaml_strings(yaml_strs, loader=DagsterRunConfigYamlLoader):
     return functools.reduce(deep_merge_dicts, yaml_dicts, {})
 
 
-def load_yaml_from_path(path: str, loader=DagsterRunConfigYamlLoader) -> object:
```

Comment:
I think we should have this return `Any` instead of `dict`, because in theory yaml doesn't need to be a dictionary.

Reply:
That's a good point - maybe we should just go with https://github.com/dagster-io/dagster/pull/10145 and skip the mypy check instead of changing the type hint. It's probably not strictly correct that we're using index access without checking in our example.